common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -1,274 +1,336 @@
 	}
+	NOT = {
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:pirate_culture}
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:northsea}
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:wastewander}	
+	}
+}
+
+can_unlock_innovation_blademaster_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:burning_blade}	
+}
+
+can_unlock_innovation_buccaneer_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:pirate_culture}
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:northsea}
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:blackwater}
+}
+
+can_unlock_innovation_brute_trigger = {
+	has_cultural_pillar = heritage_hyenaic
+}
+
+can_unlock_innovation_candleholder_trigger = {
+	has_cultural_pillar = heritage_murine
 }
-#Culture that prefers aerial and dislike everything else
-winged_culture_trigger = {
+
+can_unlock_innovation_centaur_tactics_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:centaur}
+}
+
+can_unlock_innovation_cloud_serpents_trigger = {
+	has_cultural_pillar = heritage_pandaren
+}
+
+can_unlock_innovation_dervishes_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:wastewander}	
+}
+
+can_unlock_innovation_dragon_trigger = {
 	OR = {
 		has_cultural_pillar = heritage_draconic
-		has_cultural_pillar = heritage_avianic
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:dragonmaw}
 	}
 }
-#Culture that prefers cavalry and dislike everything else
-taur_culture_trigger = {
+
+can_unlock_innovation_gladiator_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:shattered_hand}
+}
+
+can_unlock_innovation_hag_trigger = {
+	has_cultural_pillar = heritage_avianic
+}
+
+can_unlock_innovation_hexer_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:saurok}
+}
+
+can_unlock_innovation_kunchong_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:mantid}	
+}
+
+can_unlock_innovation_ogre_tactics_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:ogre}		
+}
+
+can_unlock_innovation_naga_army_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:naga}		
+}
+
+can_unlock_innovation_night_elf_aerial_trigger = {
 	OR = {
-		has_cenarius_heritage_trigger = yes
-		this = culture:tolvir
-		this = culture:nerubian
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:night_elf}
+		is_culture_or_parent_culture_trigger = {CULTURE = culture:dryad}
 	}
 }
-#Culture that prefers skirmishers and dislike cavalry and aerial
-land_skirmish_culture_trigger = {
+
+can_unlock_innovation_nerubian_tactics_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:nerubian}	
+}
+
+can_unlock_innovation_proto_drake_rider_trigger = {
+	has_cultural_pillar = heritage_vrykulic
+}
+
+can_unlock_innovation_quillboar_tactics_trigger = {
+	is_culture_or_parent_culture_trigger = {CULTURE = culture:quilboar}
```

Comment:
Check other heritages. As you see in many cases this trigger isn't needed because culture got its own heritage.
```suggestion
	has_cultural_pillar = heritage_agamagganic
```